q_learning:
  num_states: 100
  num_actions: 2
  alpha: 0.1
  gamma: 0.9
  epsilon: 0.1